aio/deploy/helm-chart/kubernetes-dashboard/README.md

```diff
@@ -0,0 +1,133 @@
+- It enables by default values for `podAnnotations` and `securityContext`, please disable them if you don't supoprt them
+- It removes `enableSkipLogin` and `enableInsecureLogin` parameters. Please use `extraEnv` instead.
+- It adds a `ProtocolHttp` parameter, allowing you to switch the backend to plain HTTP and replaces the old `enableSkipLogin` for the network part.
+- If `ProtocolHttp` is not set, it will automatically add to the `Ingress`, if enabled, annotations to support HTTPS backends for nginx-ingress and GKE Ingresses.
+- It updates all the labels to the new [recommended labels](https://github.com/helm/charts/blob/master/REVIEW_GUIDELINES.md#names-and-labels), most of them being immutable.
+
+In order to upgrade, please update your configuration to remove `clusterAdminRole` parameter and adapt `enableSkipLogin`, `enableInsecureLogin`, `podAnnotations` and `securityContext` parameters, and uninstall/reinstall the chart.
+
+## Access control
+
+It is critical for the Kubernetes cluster to correctly setup access control of Kubernetes Dashboard.
+See this [guide](https://github.com/kubernetes/dashboard/wiki/Access-control) for best practises.
+
+It is highly recommended to use RBAC with minimal privileges needed for Dashboard to run.
+
+## Configuration
+
+The following table lists the configurable parameters of the kubernetes-dashboard chart and their default values.
+
+Parameter                           | Description                                                                                                                 | Default
+------------------------------------|-----------------------------------------------------------------------------------------------------------------------------|------------------------------------------------------------------------
+`image.repository`                  | Repository for container image                                                                                              | `kubernetesui/dashboard`
+`image.tag`                         | Image tag                                                                                                                   | `v2.0.0`
+`image.pullPolicy`                  | Image pull policy                                                                                                           | `IfNotPresent`
+`image.pullSecrets`                 | Image pull secrets                                                                                                          | `[]`
+`annotations`                       | Annotations for deployment                                                                                                  | `{}`
```

Comment:
There is not description about `labels` here.
It maybe following?

```suggestion
`annotations`                       | Annotations for deployment                                                                                                  | `{}`
`labels`                       | Labels for deployment                                                                                                  | `{}`
```

Reply:
Thanks! I've added a few missing values in README, including this one.